test/player_test.rb

```diff
@@ -47,8 +47,49 @@ def test_player_can_place_ship
     @player1.add_ship(@submarine1)
     @player2.add_ship(@cruiser2)
     @player2.add_ship(@submarine2)
-
     # Tested in pry. Is working currently before auto generation and user error.
   end
 
+  def test_it_can_tell_all_ships_are_placed
```

Comment:
Thanks for testing your new methods in player_test! Great TDD and partner workflow!